packages/devtools_shared/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# NEXT
```

Comment:
4.1.0? This isn't a breaking change so a minor bump should be okay. Can you update the pubspec.yaml file too?